easybuild/easyconfigs/v/Visit/Visit-3.2.0-CrayGNU-20.11.eb

```diff
@@ -0,0 +1,94 @@
+# The downloads may fail due to different reasons. So, we have pre-packaged
+# visit with the downloaded dependencies.
+#
+# Below you will find the steps to create the SOURCELOWER_TAR_GZ file used in
+# this recipe.
+#
+# mkdir /run/user/1100/VisIt; cd /run/user/1100/VisIt
+# wget  https://github.com/visit-dav/visit/releases/download/v3.2.0/visit3.2.0.tar.gz
+# cd visit3.2.0
+# ./src/tools/dev/scripts/build_visit --console --no-visit --parallel \
+# --llvm --mesagl --vtk --makeflags -j32 --system-python --alt-python-dir /usr \
+# --openssl --qwt --qt --silo --hdf5 --cmake --icet --ospray --ispc --embree --tbb \
+# --download-only
+# cd ..
+
+# tar cfz  visit-3.2.0.tar.gz visit3.2.0
+# mkdir -p $EASYBUILD_SOURCEPATH/v/Visit
+# cp visit-3.2.0.tar.gz $EASYBUILD_SOURCEPATH/v/Visit/
+#
+# NOTES:
+# * The new visit package is named visit-3.2.0.tar.gz, not visit3.2.0.tar.gz
+# as the original file.
+# * The download-only option must not be included in the actual build.
+# * The configuration options are the same as the ones below.
+# * We are using the system-python because one cannot compile python with xalt
+# enabled.
+# * In the preconfigopts commands below, there is an additional <<< "yes".
+# This is used to accept the Qt License. This is not needed when using
+# --download-only
+# * The original intent was to compile only the server-side components, disabling the
+# use of the GUI client for loggin nodes. Yet, this also removes the cli, preventing
+# us to run python-based batch-mode scripts. Thus, we build all components [default]
+
+
+# In order to use the pre-downloaded version use the source below
+sources = ['/apps/common/UES/easybuild/sources/%(nameletterlower)s/%(name)s/%(namelower)s-%(version)s.tar.gz']
+
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
+]
+
+preconfigopts  = ' export PAR_COMPILER=$CC; '
+preconfigopts += ' export PAR_INCLUDE="-I$MPICH_DIR/include"; '
+preconfigopts += ' pushd ../visit3.2.0; mkdir -p third_party/mesagl;ln -s mesagl third_party/mesa;  module swap gcc/9.3.0; '
+preconfigopts += 'sed -i "s/{C_COMPILER/{PAR_COMPILER/g" src/tools/dev/scripts/bv_support/bv_icet.sh; '
+preconfigopts += 'sed -i "s/DMPI_COMPILER/DMPI_C_COMPILER/g" src/tools/dev/scripts/bv_support/bv_icet.sh; '
+preconfigopts += 'sed -i "0,/bin\/python/s//bin\/python3/" src/CMake/FindVisItPython.cmake; '
+preconfigopts += 'sed -i "s/PYTHON_GLOB_RESULT 0/PYTHON_GLOB_RESULT 1/g" src/CMake/FindVisItPython.cmake; '
+preconfigopts += './src/tools/dev/scripts/build_visit --console --no-visit --parallel --llvm --mesagl --vtk --makeflags -j32'
+preconfigopts += ' --system-python --alt-python-dir /usr '
+preconfigopts += ' --openssl --qwt --qt --silo --hdf5 --system-cmake --icet --ospray --ispc --embree --tbb <<< "yes" && '
+preconfigopts += ' sed -i "s/gcc TYPE/cc TYPE/g" `hostname`.cmake;  sed -i "s/g++/CC/g" `hostname`.cmake;  cp `hostname`.cmake ./src/config-site;  mkdir build; cd build; '
```

Comment:
@jfavre, just a curiosity... 
You are setting `--system-cmake`, but you are actually depending on `CMake@3.14.5`...
Can't you run this prebuild using this newer CMake...

Reply:
--system-cmake is VisIt's way to say that they will pick the cmake in the path, which is why I load our own cmake3.14.5.
I have already built it that way, and tested client-server connection and it worked well.
Thanks for looking over my shoulder